### DISCUSSION TITLE
Golang not supposed to be updatable anymore?

### DISCUSSION THREAD

**Discussion by HenrikPoulsen** [renovatebot/renovate#27288] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

37.186.1

### If you're self-hosting Renovate, select which platform you are using.

GitHub Enterprise Server

### What is your question?

For some reason I seem to no longer get updates to golang itself.
Has this changed somehow?
```
{
                 "managerData": {"lineNumber": 2},
                 "depName": "go",
                 "depType": "golang",
                 "currentValue": "1.19",
                 "datasource": "golang-version",
                 "versioning": "go-mod-directive",
                 "updates": [],
                 "packageName": "go",
                 "warnings": [],
                 "sourceUrl": "https://github.com/golang/go",
                 "registryUrl": "https://raw.githubusercontent.com/golang/website",
                 "homepage": "https://go.dev/",
                 "currentVersion": "1.22.0"
               },
```
You see that current value is `1.19` but for some reason it's setting currentVersion to `1.22.0` which I imagine makes it blind to the fact that it should recommend updating golang.
Is this expected?

### Logs (if relevant)

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
Yes, this is expected. The Go ecosystem does not recommend bumping the go mod directive just because a new version exists. What you find in go.mod files essentially means "1.19 or higher".

**Reply by HenrikPoulsen** [reply]
Ah I didn't know that. Thanks

**Reply by HenrikPoulsen** [reply]
Is it possible to override this btw?
I have some applications where I want to be on the latest update just so I get the latest language features etc, so would be nice if I can get renovate to help with keeping those updated at least

**Reply by rarkins** [reply]
Yes, if you configure rangeStrategy=bump for this package using packageRules. You also might be interested to learn about this new go.mod feature/option: https://go.dev/ref/mod#go-mod-file-toolchain
